modular_skyrat/modules/importedvendors/code/vendor_food.dm

```diff
@@ -0,0 +1,276 @@
+	tastes = list("fish?" = 2, "cheap noodles" = 2, "laziness" = 1)
+	foodtypes = VEGETABLES | NUTS | SEAFOOD
+
+/obj/item/food/vendor_tray_meal/emperor_roll
+	name = "\improper Tiziria Imports: Emperor Roll"
+	desc = "A pretty sad looking emperor roll, if you can even call it that, it seems caviar wasn't in the budget."
+	icon_state = "foodtray_emperorroll"
+	tastes = list("bread" = 2, "cheese?" = 2, "liver?" = 2, "laziness" = 1)
+	foodtypes = VEGETABLES | NUTS | DAIRY | MEAT | GORE
+
+/obj/item/food/vendor_tray_meal/mushroom_fry
+	name = "\improper Tiziria Imports: Mushroom Stirfry"
+	desc = "A mix of what was likely mushrooms too low quality to be used in making actual food, lightly fried and tossed in a plastic container together."
+	icon_state = "foodtray_shroomfry"
+	tastes = list("mushroom" = 4, "becoming rich" = 1, "laziness" = 1)
+	foodtypes = VEGETABLES
+
+/obj/effect/spawner/random/vendor_meal_sides
+	name = "random side spawner"
+	desc = "I hope I get one that actually matches my meal."
+	icon_state = "loot"
+
+/obj/effect/spawner/random/vendor_meal_sides/nt
+	name = "random nt side spawner"
+
+/obj/effect/spawner/random/vendor_meal_sides/nt/Initialize(mapload)
+	loot = list(
+		/obj/item/food/vendor_tray_meal/side,
+		/obj/item/food/vendor_tray_meal/side/crackers_and_jam,
+		/obj/item/food/vendor_tray_meal/side/crackers_and_cheese,
+	)
+	. = ..()
+
+/obj/item/food/vendor_tray_meal/side
+	name = "\improper NT-Side: Flatbread and Peanut Butter"
+	desc = "A small stack of tough flatbread, and a small spread of peanut butter for each."
+	icon_state = "foodpack_nt"
+	trash_type = /obj/item/trash/empty_side_pack/nt
+	food_reagents = list(/datum/reagent/consumable/nutriment = 5)
+	tastes = list("tough bread" = 2, "peanut butter" = 2)
+	foodtypes = GRAIN
+	hot_and_steamy = FALSE
+	custom_price = PAYCHECK_LOWER * 2.5
+
+/obj/item/food/vendor_tray_meal/side/crackers_and_jam
+	name = "\improper NT-Side: Flatbread and Berry Jelly"
+	desc = "A small stack of tough flatbread, and a small spread of nondescript berry jelly for each."
+	tastes = list("tough bread" = 2, "berries" = 2)
+	foodtypes = GRAIN | FRUIT
+
+/obj/item/food/vendor_tray_meal/side/crackers_and_cheese
+	name = "\improper NT-Side: Flatbread and Cheese Spread"
+	desc = "A small stack of tough flatbread, and a small spread of cheese for each."
+	tastes = list("tough bread" = 2, "cheese" = 2)
+	foodtypes = GRAIN | DAIRY
+
+/obj/effect/spawner/random/vendor_meal_sides/yangyu
+	name = "random yangyu side spawner"
+
+/obj/effect/spawner/random/vendor_meal_sides/yangyu/Initialize(mapload)
+	loot = list(
+		/obj/item/food/vendor_tray_meal/side/miso,
+		/obj/item/food/vendor_tray_meal/side/rice,
+		/obj/item/food/vendor_tray_meal/side/pickled_vegetables,
+	)
+	. = ..()
+
+/obj/item/food/vendor_tray_meal/side/miso
+	name = "\improper 副菜: Miso Soup"
+	desc = "This is quite literally just a plastic bag full of miso soup, remember to only open it on the side facing up."
```

Comment:
```suggestion
	desc = "This is quite literally just a plastic bag full of miso soup - remember to only open it the direction the labeling says!"
```
"The side facing up" feels vague; sure, maybe no one will ever examine this, but if they do it's better they feel immersed and left with the image of a bag with labeling, rather than wondering what side of a bag faces up

Also these are two separate statements, hence the -
You COULD use a ; but I personally feel like - helps separate a statement from a command or recommendation. Like, "Oh, that's odd - don't touch it!"